app/src/main/java/com/imashnake/animite/features/searchbar/Search.kt

```diff
@@ -35,42 +36,69 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.res.vectorResource
 import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.viewmodel.compose.viewModel
 import androidx.navigation.NavHostController
 import com.imashnake.animite.SearchQuery
+import com.imashnake.animite.features.appCurrentDestinationAsState
 import com.imashnake.animite.features.destinations.MediaPageDestination
+import com.imashnake.animite.features.navigationbar.NavigationBarPaths
 import com.imashnake.animite.type.MediaType
 import com.ramcosta.composedestinations.navigation.navigate
 import com.imashnake.animite.R as Res
 
 // TODO:
 //  - UX concern: This blocks content sometimes!
 //  - `SearchList` goes beyond the status bar.
-@OptIn(ExperimentalAnimationApi::class, ExperimentalMaterial3Api::class)
+@OptIn(
+    ExperimentalAnimationApi::class,
+    ExperimentalMaterial3Api::class,
+    ExperimentalLayoutApi::class
+)
 @Composable
-fun SearchBar(
+fun Search(
```

Comment:
Is it no longer a SearchBar? Maybe we can suffix it with what it actually is, `Search` seems vague

Reply:
It will hence be called `SearchFrontDrop`